packages/ai/src/ai/modules/task/task_tracker.py

```
@@ -0,0 +1,143 @@
+from __future__ import annotations
+
+import threading
+from dataclasses import dataclass, field
+from enum import Enum
+from time import monotonic
+from typing import Dict, Optional
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)